amida-tech/orange-api
# Feature/schedule medication reminders

Jonah's Mock Script PR should be merged in before this.
To Test:
run `npm i`
With your notification service running, schedule a medication and confirm you receive a PUSH (not Local) notification reminder for the notification (if running locally, you will only be able to receive an android notification)
